xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -3,7 +3,7 @@
     {
       "orientation" : "landscape",
       "idiom" : "tv",
-      "filename" : "splash.jpg",
+      "filename" : "splash.png",
```

Comment:
line 14 in this file must also become png

please also change the symlink in https://github.com/xbmc/xbmc/tree/master/xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage